packages/core-mobile/app/services/notifications/balanceChange/registerDeviceToNotificationSender.ts

```diff
@@ -0,0 +1,32 @@
+import Logger from 'utils/Logger'
+import AppCheckService from 'services/fcm/AppCheckService'
+
+export async function registerDeviceToNotificationSender(
+  fcmToken: string
+): Promise<{ deviceArn: string }> {
+  const appCheckToken = await AppCheckService.getToken()
+  const options: RequestInit = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'X-Firebase-AppCheck': appCheckToken.token
+    },
+    body: JSON.stringify({
+      deviceToken: fcmToken,
+      appType: 'CORE_MOBILE_APP'
+    })
+  }
+
+  const response = await fetch(
+    'https://core-notification-sender-api.avax-test.network/v1/push/register',
```

Comment:
let's use env for this

Reply:
[Extract notification api to .env](https://github.com/ava-labs/core-mobile/pull/1659/commits/50f58812ab5c64e9f78a0f82bfb60a662699d772)